Makefile

```diff
@@ -16,7 +16,7 @@ endif
 check:
 
 init:
-	./tools/cmd/openhpca_setup/openhpca_setup
+	cd ./tools/cmd/openhpca_setup && $(GOCMD) run openhpca_setup.go	
```

Comment:
@BrodyWilliams does it mean the binary is not there when you need it? The intent is to have the binary and users use the binary.

Reply:
@gvallee The original version works fine if you can "make install". The problem I ran into was, following the readme, I was attempting "make init" before anything else, which fails because the binary isn't built yet. It might be easier to just amend the Readme to clarify.

Reply:
It could also be that I just misinterpreted the Readme instructions.

Reply:
@BrodyWilliams I do not believe in misinterpretation of instructions, the doc can be either improved or the code changed to fix the problem. I suspect that the Makefile rules got out-of-order over time. I will have a look at it and will open an issue specific to it.

Reply:
I opened a separate issue and since it is fixing a problem, i am planning on merging this asap.